src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -126,4 +145,20 @@ private function formatDateTimeErrors(array $errors)
 
         return $formattedErrors;
     }
+
+    /**
+     * @param array $context
+     *
+     * @return \DateTimeZone|null
+     */
+    private function getTimezone(array $context)
+    {
+        $dateTimeZone = array_key_exists(self::TIMEZONE_KEY, $context) ? $context[self::TIMEZONE_KEY] : $this->timezone;
```

Comment:
You can use `isset` instead of `array_key_exists` here (faster as it's not a function call).